extensions/GlobalMessages/GlobalMessagesT.i18n.php

```diff
@@ -31611,100 +31774,99 @@
-Faça a fusão manual das páginas de discussão, por favor.'''",
+	'tmp-create-error' => 'Não foi possível criar o arquivo temporário.',
+	'tmp-write-error' => 'Erro ao alterar arquivo temporário.',
+	'tooltip-invert' => 'Marque esta caixa para esconder as alterações a páginas no espaço nominal selecionado (e no espaço nominal associado, se escolheu fazê-lo)',
+	'tooltip-namespace_association' => 'Marque esta caixa para incluir também o espaço nominal de conteúdo ou de discussão associado à sua seleção',
+	'talkexists' => "'''A página em si foi movida com sucesso. No entanto, a página de discussão não foi movida, uma vez que já existia uma com este título. Por favor, mescle-as manualmente.'''",
 	'thumbnail-more' => 'Ampliar',
 	'thumbnail_error' => 'Erro ao criar miniatura: $1',
-	'thumbnail-dest-create' => 'Não é possível salvar miniatura',
+	'thumbnail-dest-create' => 'Não foi possível salvar a miniatura no destino',
 	'thumbnail_invalid_params' => 'Parâmetros de miniatura inválidos',
-	'thumbnail_dest_directory' => 'Não foi possível criar o directório de destino',
+	'thumbnail_dest_directory' => 'Não foi possível criar o diretório de destino',
 	'thumbnail_image-type' => 'Tipo de imagem não suportado',
-	'thumbnail_gd-library' => 'Configuração da biblioteca GD incompleta: função $1 em falta',
-	'thumbnail_image-missing' => 'Ficheiro em falta: $1',
-	'tooltip-pt-userpage' => 'A sua página de utilizador',
-	'tooltip-pt-anonuserpage' => 'A página de utilizador para o endereço IP que está a usar',
-	'tooltip-pt-mytalk' => 'A sua página de discussão',
-	'tooltip-pt-anontalk' => 'Discussão sobre edições feitas a partir deste endereço IP',
-	'tooltip-pt-preferences' => 'Configuração dos comportamentos que prefere da wiki',
-	'tooltip-pt-watchlist' => 'Lista de mudanças nas páginas que está a vigiar',
-	'tooltip-pt-mycontris' => 'Lista das suas contribuições',
-	'tooltip-pt-login' => 'É encorajado a autenticar-se, apesar de não ser obrigatório.',
-	'tooltip-pt-anonlogin' => 'É encorajado a autenticar-se, apesar de não ser obrigatório.',
-	'tooltip-pt-logout' => 'Terminar esta sessão na wiki',
+	'thumbnail_gd-library' => 'Configuração da biblioteca GD incompleta: função $1 não encontrada',
+	'thumbnail_image-missing' => 'Arquivo aparentemente inexistente: $1',
+	'tooltip-pt-userpage' => 'Sua página de usuário',
+	'tooltip-pt-anonuserpage' => 'A página de usuário para o ip com o qual você está editando',
+	'tooltip-pt-mytalk' => 'Sua página de discussão',
+	'tooltip-pt-anontalk' => 'Discussão sobre edições deste endereço de ip',
+	'tooltip-pt-preferences' => 'Suas configurações',
+	'tooltip-pt-watchlist' => 'Lista de alterações nas páginas que você está monitorando',
+	'tooltip-pt-mycontris' => 'Listagem de suas contribuições',
+	'tooltip-pt-login' => 'Você é encorajado a autenticar-se, apesar disso não ser obrigatório.',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "Você é encorajado a autenticar-se, apesar disso não ser obrigatório." 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)